SDK/include/values.hpp

```diff
@@ -62,33 +66,33 @@ constexpr int MAX_GAMETEXT_STYLES = 16;
 
 enum Key
 {
-	ACTION = 1,
-	CROUCH = 2,
-	FIRE = 4,
-	SPRINT = 8,
-	SECONDARY_ATTACK = 16,
-	JUMP = 32,
-	LOOK_RIGHT = 64,
-	HANDBRAKE = 128,
+	ACTION = 0b1 << 0,
```

Comment:
The `0b` is logically correct, but since everything is just `0b1` I think not using `1` is just visual clutter.